city/router.py

```diff
@@ -0,0 +1,53 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from dependencies import get_db_session
+from . import crud, schemas
+
+
+router = APIRouter()
+
+
+@router.post("/cities/", response_model=schemas.City)
+async def create_city(
+    city: schemas.CityCreate,
+    db: AsyncSession = Depends(get_db_session),
+):
+    return await crud.create_city(city=city, db=db)
+
+
+@router.get("/cities/", response_model=list[schemas.City])
+async def get_all_cities(db: AsyncSession = Depends(get_db_session)):
+    return await crud.read_cities(db=db)
+
+
+@router.get("/cities/{city_id}", response_model=schemas.City)
```

Comment:
```suggestion
@router.get("/cities/{city_id}/", response_model=schemas.City)
```

fix in all similar cases